asyncio_run_in_process/run_in_process.py

```diff
@@ -192,12 +221,10 @@ def open_in_process(
 
     sub_proc = await asyncio.create_subprocess_exec(
         *command,
-        # stdin=subprocess.PIPE,
-        # stdout=subprocess.PIPE,
-        # stderr=subprocess.PIPE,
-        pass_fds=(child_r, child_w),
+        # mypy doesn't recognize that loop can be `None`
+        loop=loop,  # type: ignore
```

Comment:
Do we really need to add the loop argument here? It has been deprecated in 3.8

Reply:
Nope! didn't see that it was deprecated.  Will remove.